### PR TITLE
Add COOP: "Coopdevs"

### PR DESCRIPTION
**Pull request by saimonmoore**
Cooperative organization in Barcelona, Catalonia created to help promote
social change, especially in the sector of food consumption (See
http://www.katuma.org/) (Open Food Network) but also contributes substantially to the
[Timeoverflow](https://twitter.com/timeoverflow) time bank org and
works closely with [a large range of non-profits/social change organizations](http://coopdevs.org/) in Catalonia.

You can contact @enricostano / @sauloperez for any additional info (or verification) you may need.

(I'm just a friend helping them out when I can)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +79,7 @@ Coop | Business Areas | Region/Country | Notes
 [Tecso](http://www.tecso.coop/en/) | Development, Information services | Rosario, Argentina | Founded 2002; 150+ associates
 [Vulk Coop](http://vulk.coop) | Design, Development | Austin, TX, USA |
 [Webarchitects Co-operative](https://www.webarchitects.coop) | Hosting, Sysadmin | Sheffield, UK | Multi-stakeholder, including workers, organisations and investors |
+[Coopdevs](http://coopdevs.org/) | Development | Barcelona, Catalonia | Focusing on non-profit & grassroots social change projects e.g. Open Food Network amongst others |
```
Comment (enricostano) [inline README.md]: Maybe we should specify that we're a non profit association and a workers cooperative.

Comment (saimonmoore) [reply README.md]: Agreed! Fixed. (The non-profit bit is a bit redundant but I added it anyway.)

**Comment by hng** [comment]
Awesome! Could you insert your coop into the listing in alphabetical order?

**Comment by saimonmoore** [comment]
Yep fixed it locally but looks like there are issue with gh and I can't push

**Comment by hng** [comment]
@saimonmoore Thanks!

**Pull request merged**